components/composed/user/UserList/UserList.tsx

```diff
@@ -45,7 +44,6 @@ function UserList<T extends OperationType>({
     <ModelListPage<T, UserListFragment, UserNode>
       modelName="user"
       columns={columns}
-      viewOptions={[DataViewOptions.table]}
```

Comment:
Because so many lists had just `table` as their view option, I've set that to be the default. It was rare to see all options used (collections, items)

Reply:
+1